clang/tools/clang-offload-bundler/ClangOffloadBundler.cpp

```diff
@@ -792,7 +804,62 @@ class ObjectFileHandler final : public FileHandler {
       ObjcopyArgs.push_back(SS.save(Twine("--add-section=") +
                                     SIZE_SECTION_PREFIX + TargetNames[I] + "=" +
                                     *SizeFileOrErr));
+
+      if (I != HostInputIndex) {
```

Comment:
It might be forth moving new code to a separate function to make it more readable. The scope of for-loop at line 780 is quite big to comprehend.

Reply:
Done